sugargame.canvas.patch

```diff
@@ -0,0 +1,18 @@
+diff --git a/sugargame/canvas.py b/sugargame/canvas.py
+index 90856ca..a506fdf 100755
+--- a/sugargame/canvas.py
++++ b/sugargame/canvas.py
+@@ -42,8 +42,11 @@ class PygameCanvas(Gtk.EventBox):
+         widget.props.window.set_cursor(None)
+ 
+         # Confine the Pygame surface to the canvas size
+-        r = self.get_allocation()
+-        self._screen = pygame.display.set_mode((r.width, r.height),
++        from sugarapp.helpers import PrimaryMonitor
```

Comment:
Instead of patching sugargame, you can simply use PrimaryMonitor to set the request size of the canvas from the app code. Check how it is done in Physics.

Reply:
Changed it to the activity.py file, i have to check it out